src/Struct/FindologicProduct.php

```diff
@@ -762,14 +762,30 @@ protected function setOrdernumberByProduct(ProductEntity $product): void
     protected function setCategoriesAndCatUrls(): void
     {
         $productCategories = $this->product->getCategories();
-        if ($productCategories === null || empty($productCategories->count())) {
+        $productVariants = $this->product->getChildren();
```

Comment:
The naming `$productCategories` makes sense here, as this only includes the categories assigned directly to the product. However only a product can have any variants, therefore the naming `$productVariants` is too much information. Calling it `$variants` would be sufficient.